config.ts

```diff
@@ -9,8 +9,10 @@ export enum DaoId {
 
 export const config = {
   [DaoId.Example]: {
-    name: 'Example Governor',
+    name: 'Example',
+    id: DaoId.Example,
```

Comment:
(arbitrary place to leave comment)

given the new routing, we might want to prefer urls with an identifying string in them (aka */pooltogether/bridge*) which is easily done via string enum for DaoId type:

i.e. 
```
enum DaoId {
  Example = "pooltogether",
  ExampleComp = "example-comp"
}
```

if you agree, happy to call it follow up and not block merge

Reply:
My one worry with that scheme is we can run into future problems if names aren't unique. Although, I don't see that being an issue anytime soon. We could do `/[id]/[name]/bridge`?

Reply:
Happy to flesh out more in an issue as well https://github.com/ScopeLift/l2-flexible-voting-frontend/issues/115